test/functionalTest/cases/715_forward_xauth_token_to_cprs/propagate_xauth_token_for_update_and_query_standard_ops.test

```diff
@@ -238,92 +238,96 @@ Date: REGEX(.*)
 06. Dump accumulator to see X-Auth headers (or not)
```

Comment:
If you look at them in more detail, you will probably see that the register was made in JSON.
XML is not the default format in this case, but the format that was used as 'outFormat' (or 'Accept') when the registration was made.